ddtrace/internal/sampling.py

```diff
@@ -1,18 +1,29 @@
 import re
+from typing import Optional
 from typing import TYPE_CHECKING
 
+from ddtrace.constants import _SINGLE_SPAN_SAMPLING_MAX_PER_SEC
+from ddtrace.constants import _SINGLE_SPAN_SAMPLING_MECHANISM
+from ddtrace.constants import _SINGLE_SPAN_SAMPLING_RATE
+from ddtrace.internal.glob_matching import GlobMatcher
 from ddtrace.internal.logger import get_logger
 
+from .rate_limiter import RateLimiter
+
 
 log = get_logger(__name__)
 
 if TYPE_CHECKING:
     from typing import Dict
-    from typing import Optional
     from typing import Text
 
     from ddtrace.context import Context
 
+    from ..span import Span
+
+KNUTH_FACTOR = 1111111111111111111
+MAX_SPAN_ID = 2 ** 64
```

Comment:
```suggestion
MAX_SPAN_ID = 1 << 64
```
Since the max span id is function of the number of bits using a bitshift operator might be a little easier to read.

Reply:
I wanted to follow what we have in `sampler.py` https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/sampler.py#L47 but let me know if you think I should still change.